src/async.js

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash/fp'
+import Promise from 'bluebird'
+
+export let asyncMap = _.curry((fn, obj) => Promise.map(obj, fn))
```

Comment:
If we are just doing a curry here and flipping the param order of the bluebird method, how can we preserve the options param that Bluebird.map can take, to control concurrency for example. maybe have asyncMapOpt that does _.curry(opts, fn, obj) and your asyncMap becomes asyncMapOpt({}) or something like that

Reply:
exactly why this really belongs as a bluebird-fp lib - we're really just currying and re-arging the bluebird stuff here

Reply:
But so this `bluebird-fp` should have `asyncFlow`? isn't it better to call this separate library something like `async-fp`??

Reply:
just the ones that are just rearging/currying bluebird methods